Packages/com.walletconnect.web3modal/Runtime/Controllers/AccountController.cs

```diff
@@ -87,9 +87,11 @@ public async Task InitializeAsync(ConnectorController connectorController, Netwo
             _connectorController = connectorController ?? throw new ArgumentNullException(nameof(connectorController));
             _networkController = networkController ?? throw new ArgumentNullException(nameof(networkController));
             _blockchainApiController = blockchainApiController ?? throw new ArgumentNullException(nameof(blockchainApiController));
-            
+
+#if !UNITY_WEBGL || UNITY_EDITOR
             _connectorController.AccountConnected += ConnectorAccountConnectedHandler;
             _connectorController.AccountChanged += ConnectorAccountChangedHandler;
+#endif
```

Comment:
Are these features not supported on WebGL?

Reply:
These event handler update ENS name, avatar, and account balance in native Unity UI. In WebGL `@web3modal/wagmi` takes care of that, so these handler were just making duplicate requests to the blockchain api.